core/src/main/java/io/undertow/server/protocol/http2/Http2ServerConnection.java

```diff
@@ -110,6 +111,7 @@ public Http2ServerConnection(Http2Channel channel, Http2StreamSourceChannel requ
         originalSourceConduit = new StreamSourceChannelWrappingConduit(requestChannel);
         this.conduitStreamSinkChannel = new ConduitStreamSinkChannel(responseChannel, originalSinkConduit);
         this.conduitStreamSourceChannel = new ConduitStreamSourceChannel(channel, originalSourceConduit);
+        this.createdTimestamp = System.currentTimeMillis();
```

Comment:
Is this correct? If memory serves, a new `Http2ServerConnection` object is created for each request, so the createdTimestamp will be set at the same time as `HttpServerExchange.requestStartTime` (which uses the high resolution nanoTime, and is default disabled for performance).

Reply:
OK. Looks like the semantics of the Connection changed in Http2: the equivalent to that of Http in Http2 is `Http2Channel`. And you're right that a new `Http2ServerConnection` object is created for each request, however, this is still the connection created timestamp, just not useful anymore.

Reply:
@eagleshine does that mean that this PR won't work the way you intended it to? If it is no longer valid, please close it. If that's not the case, please let me know